### ISSUE TITLE
Pick a Go framework

### ISSUE THREAD

**Issue by revanthkolli** [revanthkolli/feather#7]

**Comment by revanthkolli** [comment]
Not using a framework since this will be a basic API. Since we're thinking about GraphQL, might not need a REST / http framework either.

**Comment by kalyanaramansanthanam** [comment]
if we are going with GraphQL, still need to pick whether we want to use a simple HTTP GraphQL server like  https://github.com/graphql-go/handler or Relay https://relay.dev/

This blog/tutorial is good in putting all the pieces together ->  https://wehavefaces.net/learn-golang-graphql-relay-1-e59ea174a902

**Comment by kalyanaramansanthanam** [comment]
reading this blog makes me think, we should use relay since it tightly binds with react https://artsy.github.io/blog/2019/04/10/omakase-relay/

**Comment by kalyanaramansanthanam** [comment]
lemme `go` through the tutorials in the blog for relay tomorrow and give my feedback on this

**Comment by revanthkolli** [comment]
Yup Relay seems good for the React client side, we would still need a separate library for the Go server side right?
